scripts/flink-image-install.sh

```diff
@@ -8,4 +8,19 @@
 #     http://www.apache.org/licenses/LICENSE-2.0
 set -ex
 ROOT_DIR=$(readlink -f $(dirname $0)/..)
-kubectl apply -f ${ROOT_DIR}/flink-image/ClusterFlinkImage.yaml
+source ${ROOT_DIR}/scripts/env.sh
+# Set the docker registry with desa installe
+echo sdp install script $SDP_INSTALL_SCRIPT
```

Comment:
Remove this line. `set -ex` at the top will echo all commands.

Reply:
removed set -ex